pygsti/models/model.py

```diff
@@ -610,6 +611,56 @@ def num_params(self):
         """
         self._clean_paramvec()
         return len(self._paramvec)
+    
+    @property
+    def parameter_labels(self):
+        """
+        A list of labels, usually of the form `(op_label, string_description)` describing this model's parameters.
+        """
+        self._clean_paramvec()
+        return self._paramlbls
```

Comment:
So I thought the merge would catch this but apparently not. If the merge didn't catch it, then I thought the tests would complain about a double property definition. But it looks like neither happened...
But #488 also introduced a parameter_labels, defined on 493 above. Should these be combined like so?

```
@property
    def parameter_labels(self):
        """
        A list of labels, usually of the form `(op_label, string_description)` describing this model's parameters.
        """
        self._clean_paramvec()
        return self._ops_paramlbls_to_model_paramlbls(self._paramlbls)
```

Reply:
That is strange... but they should indeed be combined as you did above.

Reply:
Mkay, I will do so and merge after that unless you had other blocking issues?